### PR TITLE
Set row header label's default to "middle"

### PR DESCRIPTION
**Pull request by melissatdiamond**
Fix issue #3891

### CODE REVIEW

**Comment by kanitw** [comment]
Rebase and compile examples again?

File: examples/compiled/bar_grouped.vg.json [hunk 1]
```diff
@@ -144,7 +144,14 @@
                 },
                 "offset": 10,
                 "orient": "top",
-                "style": "guide-label"
+                "style": "guide-label",
+                "encode": {
```
Comment (kanitw) [inline examples/compiled/bar_grouped.vg.json]: We can actually use `title` directive's `baseline` property.
However, it seems like this is not the problem introduced by this PR. Thus, let's refactor this in a separate PR.  (Perhaps as a part of https://github.com/vega/vega-lite/pull/3888?)

**Comment by melissatdiamond** [comment]
It seems like the only example that is not working is the facet bullet example? I'm not sure why that would be the case.

**Comment by kanitw** [comment]
Test still failing.

**Comment by kanitw** [comment]
Please:

1) Fix this

image

2) squash the second rebuild examples into the commit the make changes.

3) As stated in #3891, add the facet_bullet example to the gallery. (Otherwise this issue won't really fix #3891)

**Comment by domoritz** [comment]
Ready @kanitw

**Comment by kanitw** [comment]
Hmm there is now a conflict.  Rebase, re-compile example, and squash my commit to be a part of the "Add facet_bullet to the gallery" commit?

**Comment by domoritz** [comment]
@melissatdiamond Can you squash the last two commits?

**Comment by domoritz** [comment]
@kanitw Please merge.

**Pull request merged**